components/avatar/avatar.css

```diff
@@ -0,0 +1,25 @@
+.avatar {
+    height: var(--sp-4);
+    width: var(--sp-4);
+    border-radius: 50%;
```

Comment:
you can convert % into rem here as well

Reply:
Thanks, it's just my preference to use % instead of rem while making a circle.